src/project-ontology.ttl

```diff
@@ -0,0 +1,135 @@
+@prefix : <http://bibsysdev.github.io/src/project-ontology#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix nva: <http://bibsysdev.github.io/src/nva> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix schema: <https://schema.org/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+:Project a rdfs:Class ;
+  rdfs:comment "An organised effort designed to achieve a particular goal" ;
+  rdfs:subClassOf schema:Project .
+
+:Identifier a rdfs:Class ;
+  rdfs:comment "An identifier that uniquely identifies something" .
+
+:CristinIdentifier a rdfs:Class ;
+  rdfs:comment "An identifier that uniquely identifies the type of the domain scope in Cristin" ;
+  rdfs:subClassOf :Identifier .
+
+:identifier a rdf:Property ;
+  rdfs:comment "Property relating a project to its identifiers" ;
+  rdfs:domain :Project ;
+  rdfs:range :Identifier .
+
+:value a rdf:Property ;
+  rdfs:comment "A value associated with an Identifier" ;
+  rdfs:domain :Identifier ;
+  rdfs:range xsd:string .
+
+:title a rdf:Property ;
+  rdfs:comment "The name of a thing" ;
+  rdfs:domain :Project, :Grant ;
+  rdfs:range xsd:string .
+
+:language a rdf:Property ;
+  rdfs:comment "The main language of the project" ;
+  rdfs:domain :Project .
+
+:alternativeTitle a rdf:Property ;
+  rdfs:comment "An alternative or translated name for a project" ;
+  rdfs:domain :Project ;
+  rdfs:range rdf:langString .
+
+:startDate a rdf:Property ;
```

Comment:
Since we currently cannot get history of contributors from Cristin API we can not implement startDate and endDate for Contributor yet, but might come at a later time as Cristin does record these changes internally